### DISCUSSION TITLE
Any way to decrease the crop increments?

### DISCUSSION THREAD

**Discussion by trych** [rsjaffe/MIDI2LR#729] [Problems]
Hi there!

While it is possible to set the limits (and therefore tune the increments) of properties like Exposure and Temperature, I mainly have an issue with the crop limits being to extreme and therefore having no fine enough control over the cropping.

As I need this mainly for cropping the very edges of analog scans, it would be really helpful for me, if I could set the limits to something like 10% instead of 100% for cropping. Is there any way to do this?

Thanks!

**Comment by illuminancephoto** [comment]
You'll want the crop commands assigned to a control that sends relative values.  Right click the command in the MIDI2LR window to get the "Adjust CC dialog" window and make sure you've picked whichever relative mode your controller is sending.  Now experiment with the resolution setting to adjust how fast changes happen when you move the control.

**Comment by trych** [comment]
I have a Behringer XTouch mini, I think this one does not have relative controls, or does it.

When I change the modes in the "Adjust CC dialog" window to anything but Absolute, I get completely erratic behaviours, the crop does not happen or jumps all over the place. Also, the controller is already set to its maximum of 127, I guess I cannot make it more "fine grained" then this?

So, I probably would need to change the limits of the crop property itself somewhere. Any idea if that's possible?

**Reply by illuminancephoto** [reply]
Right, 0-127 is the best you can do, which is why you'll want relative values.  Any of the encoders can be switched to relative mode in the X-Touch Editor software.  It will only run on windows so if you use mac you'll have to find a windows machine to run it on.  In the minimum value setting for any encoder you'll find relative1, relative2, and relative3 as options.  I don't recall which is which but they correspond to two's complement etc.  [https://github.com/rsjaffe/MIDI2LR/wiki/MIDI-Controller-Setup#controllers-that-send-relative-values-or-have-unusual-ranges](url). Get your X-Touch and MIDI2LR using the same type of relative first.  Then use the resolution setting in MIDI2LR to set how much encoder motion it takes to change the crop.  A particular encoder rotation could give you a 1% change or 10%, depending on how the resolution is set.

**Reply by trych** [reply]
Oh thank you so much, this works! I changed the settings to relative with the editor on Windows and then set the resolution to 511. This works really well. I tested all the different relative options, they all seemed to work for me, so I set all the knobs that I needed to relative 1.

Thanks again!
